iib/workers/dogpile_cache.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import os
+import threading
+
+from celery import current_task
+from dogpile.cache import make_region
+
+from iib.workers.config import get_worker_config
+
+
+def generate_cache_key(fname, *arg, **kw):
```

Comment:
Why not have this as a staticmethod inside the class?

Reply:
I would not suggest to putting these inside the class that is declared below (DogpileNamespace), because these functions are not related to namespace.
I am not sure what will be the benefit to putting those functions to class? When we do not have to store any context.

Reply:
Correct, I think you're right. We have two options here, either we could name the class as DogpileCache and move these as @staticmethod or keep it as it is. Your call. :)

Reply:
I am voting for keeping it this way unless other reviewers disagree. 
Thank you.